rest/route/project_test.go

```diff
@@ -776,3 +776,20 @@ func TestDeleteProject(t *testing.T) {
 	resp = pdh.Run(ctx)
 	assert.Equal(t, http.StatusBadRequest, resp.Status())
 }
+
+////////////////////////////////////////////////////////////////////////
+//
+// Tests for PUT /rest/v2/projects/value
+
+func (s *ProjectPutSuite) TestNotSureIfThisRight() {
+	ctx := context.Background()
+	json := []byte(
+		`{
+				"toReplace": "a",
+				"replacement": "z"
+		}`)
+
+	req, _ := http.NewRequest("PUT", "http://example.com/api/rest/v2/projects/value", bytes.NewBuffer(json))
+	err := s.rm.Parse(ctx, req)
+	s.NoError(err)
```

Comment:
Can we please somehow check to make sure that the value was parsed correctly (that h.replaceVars contains the right values) and maybe if it's practical, check that it was replaced?

Reply:
im having trouble with this and im not sure why because i can request it locally on postman fine.

Reply:
Perhaps what we can do is insert a project and some project vars into the local db, run your endpoint, then fetch that project to make sure the vars are indeed the new ones. You can take a look at TestPatchTriggerAliases() to see how the db operations would be set up.

Reply:
fixed!